packages/lit-node-client-nodejs/src/lib/helpers/get-signatures.ts

```diff
@@ -0,0 +1,255 @@
+import { LIT_CURVE, LIT_ERROR } from '@lit-protocol/constants';
+import { combineEcdsaShares } from '@lit-protocol/crypto';
+import {
+  logErrorWithRequestId,
+  logWithRequestId,
+  mostCommonString,
+  throwError,
+} from '@lit-protocol/misc';
+import { SigResponse, SigShare } from '@lit-protocol/types';
+import { joinSignature } from 'ethers/lib/utils';
+
+export const getFlattenShare = (share: any): SigShare => {
+  // flatten the signature object so that the properties of the signature are top level
+  const flattenObj = Object.entries(share).map(([key, item]) => {
+    if (item === null || item === undefined) {
+      return null;
+    }
+
+    const typedItem = item as SigShare;
+
+    const requiredShareProps = [
+      'sigType',
+      'dataSigned',
+      'signatureShare',
+      'shareIndex',
+      'bigR',
+      'publicKey',
+    ];
+
+    const requiredSessionSigsShareProps = [
+      ...requiredShareProps,
+      'siweMessage',
+    ] as const;
+
+    const requiredSignatureShareProps = [
+      ...requiredShareProps,
+      'sigName',
+    ] as const;
+
+    const hasProps = (props: any) => {
+      return [...props].every(
+        (prop) =>
+          typedItem[prop as keyof SigShare] !== undefined &&
+          typedItem[prop as keyof SigShare] !== null
+      );
+    };
+
+    if (
+      hasProps(requiredSessionSigsShareProps) ||
+      hasProps(requiredSignatureShareProps)
+    ) {
+      const bigR = typedItem.bigR ?? typedItem.bigr;
+
+      typedItem.signatureShare = (typedItem.signatureShare ?? '').replaceAll(
+        '"',
+        ''
+      );
+      typedItem.bigR = (bigR ?? '').replaceAll('"', '');
+      typedItem.publicKey = (typedItem.publicKey ?? '').replaceAll('"', '');
+      typedItem.dataSigned = (typedItem.dataSigned ?? '').replaceAll('"', '');
```

Comment:
Do we do this in more than just here? thinking this might be better as a separate helper for sanitizing signatures.

Reply:
Yea, I think so. created a ticket for this https://linear.app/litprotocol/issue/LIT-3082/[js-sdk]-separate-flatten-shares-helper-function